src/clj/ataru/applications/application_store.clj

```diff
@@ -204,8 +204,17 @@
 (defn get-application [application-id]
   (unwrap-application (first (exec-db :db yesql-get-application-by-id {:application_id application-id}))))
 
-(defn get-latest-application-by-key [application-key]
-  (unwrap-application (first (exec-db :db yesql-get-latest-application-by-key {:application_key application-key}))))
+(defn get-latest-application-by-key [application-key organization-oids]
+  (println (str "get-latest-application-by-key: " application-key))
```

Comment:
Accidentally left `(println)` that was used during development to the production level code. Removed completely.